app/views/layouts/application.html.erb

```diff
@@ -23,7 +23,11 @@
       <header class="main-header">
         <div class="toolbar">
           <div class="toolbar__left">
-            <h1 class="toolbar__item main-header__title"><a class="main-header__site-title" href="/">ClientComm</a><span class="label label--magenta">Beta</span></h1> 
+            <h1 class="toolbar__item main-header__title">
+              <a class="main-header__site-title" href="/">ClientComm</a>
+              <span class="user-phone-number is-mobile-hidden--inline"> | <%= @user_phone_number %></span>
```

Comment:
The *Beta* label should be connected to the *ClientComm* title; let's put the number to the right of the *Beta* instead.